barney.yaml

```diff
@@ -7,12 +7,19 @@ images:
       - image: barney.ci/alpine%pkg/coreutils
 
   minimal-source:
+    no-create-mountpoints: true
     units:
-      - floor: .%build-floor-alpine
-        build: |
+      - build: |
           mkdir -p /dest/usr/lib/python3.11/site-packages/pybsn/
           cp pybsn/__init__.py /dest/usr/lib/python3.11/site-packages/pybsn/
 
+  minimal-source-py39:
```

Comment:
We are looking to get away from platform specific references.  Would https://github.com/bigswitch/pybsn/blob/main/barney.yaml#L16 work?  i.e. copy to the correct location in the consumer or add to PYTHONPATH.

Reply:
It should have the "no-create-mountpoints: true" added.

Reply:
Yeah should do. I'll leave it to you folks!